### DISCUSSION TITLE
[FEATURE REQUEST] Approve Comments

### DISCUSSION THREAD

**Discussion by atomicpoet** [mastodon/mastodon#21447] [General]
When making a thread, allow for comments to be received but only appear *if* the OP approves them to be seen. 

This is useful for:

1. Time-sensitive conversations
2. Wanting to verify accounts before comments are made public
3. Generally slowing down damage from trolls

**Comment by MattWilcox** [comment]
Food for thought: This could be somewhat open for a negative impact on the overall network.

It allows the creation of an echo chamber by allowing someone to only publicise responses that agree with them. This can falsely elevate an idea in the wider public consciousness.

Personally, I would prefer this option to not make it into production for that reason. One of the powerful features of Mastodon is that it is normal conversations without features that facilitate dog-piling or skewed perspectives through censorship - which this would be capable of being used as.
